### ISSUE TITLE
Reflection TS requires resolution of "concepts in any context" issue 29

### ISSUE THREAD

**Issue by Axel-Naumann** [cplusplus/reflection-ts#55]
Reflection TS as spelled by your rebase requires the resolution of "concepts in any context" issue 29 http://cplusplus.github.io/concepts-ts/ts-complete.html#29

How do we deal with that? Shall we incorporate the resolution to issue 29 into the Reflection TS, or is a resolved issue implicitly included in our base document "Concepts TS"?

**Comment by camio** [comment]
I don't think it matters much in practice so my preference would be to leave it as is.

**Comment by camio** [comment]
AN: We use concept in "boolean context" quite a lot, in the TS document
("Expression<T> is true if and only if T reflects an expression (Clause 8).")
and that was not a thing in the original Concepts-TS - but there was an issue
on it and that got resolved as part of the merge of the TS into the WD. As such
we can probably use it here. Hubert, what's your take?

HT: I think trying to apply the interpretation that this talks about "concepts
anywhere" is not helpful. Only mild squinting is needed to say that this means
the concept is satisfied iff <condition>.

**Comment by camio** [comment]
Leaving open as a post-publication issue.
